README.md

```diff
@@ -40,146 +40,6 @@ worker:
   image: your/invenio-image
 ```
 
-The following table lists the configurable parameters of the `invenio` chart and their
-default values, and can be overwritten via the helm `--set` option.
-
-### General
-
- Parameter | Description   | Default
------------|---------------|----------------------
- `host`    | Your hostname | `yourhost.localhost`
-
-### Invenio
-
- Parameter                                           | Description                                                                                                           | Default
------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|-----------------------
- `invenio.secret_key`                                | The Invenio secret key used for encryption; **set it**!                                                               | `secret-key`
- `invenio.security_login_salt`                       | The Invenio salt value, used when generating login links/tokens; **set it**!                                          | `security_login_salt`
- `invenio.csrf_secret_salt`                          | The Invenio salt value, used when generating login CSRF tokens; **set it**!                                           | `csrf_secret_salt`
- `invenio.init`                                      | Whether to initiate database, index and roles                                                                         | `false`
- `invenio.default_users`                             | If set, create users identified by email:password on install (only works if init=true)                                | `nil`
```

Comment:
comment: I would link this PR in a docs-related issue, since by removing the table we have some information loss (still some of the values and their descriptions are outdated).

Reply:
Alright. I'll create an issue for it!

Reply:
Done. Mark this comment as resolved if you think the Issue looks good. Also feel free to comment on it if you want to add any additional context to it.